backend/src/main/java/com/huellapositiva/domain/model/entities/ESAL.java

```diff
@@ -40,4 +51,23 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id);
     }
+
+    public static ESAL parseJpa(JpaESAL esal) {
+        URL logoUrl;
+        try{ logoUrl = new URL(esal.getLogoUrl()); }
+        catch (MalformedURLException ex) { logoUrl = null; }
```

Comment:
This should not happen if we validate the inputs correctly.

Remove this and if it breaks something think how that should have been validated.

Reply:
Fixed